### ISSUE TITLE
clarification on implicit solvent parameters

### ISSUE THREAD

**Issue by EqualAPriori** [openmm/openmm#3708]
Hi all,

I had some questions regarding the [tutorial](http://docs.openmm.org/latest/userguide/application/02_running_sims.html#force-fields) (and perhaps [issue 3214](https://github.com/openmm/openmm/pull/3214)). Apologies if I missed some other relevant thread or resource!

I'm currently making some peptides using AmberTools tleap, and creating rst7 and parm7 files (e.g. with ff96). I can then run implicit simulations with the parm7 file, for example following [this tutorial](https://parmed.github.io/ParmEd/html/omm_amber.html). As far as I am aware, when I'm making the parm7 file I don't specify any IGB settings, and this is handled by setting OpenMM settings when creating the system.

I want to contrast this approach with loading forcefields that ship with OpenMM. In particular, in the [OpenMM user guide](http://docs.openmm.org/latest/userguide/application/02_running_sims.html#force-fields), it states that for IGB2,5 (the OBC models) that 

> Note that the GBSA-OBC parameters in these files are those used in TINKER.[[27](http://docs.openmm.org/latest/userguide/zbibliography.html#id55)] They are designed for use with Amber force fields, but they are different from the parameters found in the AMBER application.

Does this mean that using the parm7 files generated by AmberTools with IGB2/5 is wrong, and that the best practice is to use the OpenMM XML files if using IGB 2,5? Presumably because there are some modifications therein that deviate from Amber's parm7 parameters?

Thanks!

**Comment by peastman** [comment]
You're looking at the "Older Force Fields" section.  When using more modern force fields like Amber14 and CHARMM36, you include the files described in [this section](http://docs.openmm.org/latest/userguide/application/02_running_sims.html#implicit-solvent).  They should match the values used by tleap.  The table specifies which igb value each file corresponds to.

**Comment by EqualAPriori** [comment]
That's right -- I was looking to check out/test an older combination, ff96+igb5, which in some reports was comparable to the more modern ff14SBonlysc + igb8.

In particular, I was curious what differs between:
```python
ala5_gas = parmed.load_file('ala5_gas.parm7', 'ala5_gas.rst7')
system = ala5_gas.createSystem(nonbondedMethod=app.NoCutoff,
                               constraints=app.HBonds, implicitSolvent=app.OBC2,
                               implicitSolventSaltConc=0.1*u.moles/u.liter,
)
```
(where `ala5_gas.parm7` is obtained from leap with ff96), vs. directly including the forcefield xml files following the link you gave. 

Thanks!

**Comment by jchodera** [comment]
> As far as I am aware, when I'm making the parm7 file I don't specify any IGB settings, and this is handled by setting OpenMM settings when creating the system.

The `parm7` file you generate with`tleap` will include GBSA parameters that are read and used by OpenMM. The `implicitSolvent=app.OBC2` argument to `createSystem` will just select the fucntional form---I don't believe it will change the parameters for you. This means you will need to include the appropriate `set default PBRadii <value>` in the `tleap` input in order to ensure GBSA parameters appropriate for the GBSA functional form model you want to use (here, OBC2) are set.

The [Amber 2022 manual](https://ambermd.org/doc12/Amber22.pdf) recommends:
![image](https://user-images.githubusercontent.com/3656088/180352509-6f0a904a-f3b8-4116-a3ab-c505a7d37a9f.png)
where OBC2 corresponds to the OBC model II (`igb=5`):
![image](https://user-images.githubusercontent.com/3656088/180352638-b1b6ff01-9066-4773-8003-2806848e0048.png)
So for OBC2, you would want to include 
```
set default PBRadii mbondi2
```
in your `tleap` input.

**Comment by EqualAPriori** [comment]
Thanks! That's super helpful, I missed that part of the amber manual. Is the idea that using, say "amber96_obc.xml" effectively updates the PBradii value from the default one? And is that the reason for the following comment from the openmm manual:

> Note that the GBSA-OBC parameters in these files are those used in TINKER.[[27](http://docs.openmm.org/latest/userguide/zbibliography.html#id55)] They are designed for use with Amber force fields, but they are different from the parameters found in the AMBER application.

**Comment by peastman** [comment]
Back when those older force fields were created, GBSA was still a new method and not well standardized.  I'm not sure of the exact circumstances that led to different programs using different parameters.  I don't know if any really thorough analysis was ever run comparing the accuracy of the different sets.  ff96 is a really old force field that has been considered out of date for a long time.  The situation is better with modern force fields.

**Comment by membiophys** [comment]
Hi !
I have a question about the non polar contribution (proportional to SASA) in the implicit solvent models implemented in OpenMM. I do not see any mention to it in documentation or tutorials, so I wonder if it is at all included, and if yes, how to use it.
Thank you in advance for your time and comments

**Comment by peastman** [comment]
Can you open a new issue for your question rather than asking on an unrelated issue?

**Comment by membiophys** [comment]
Thank you. I just opened new issue  about that: #4053
